samples/meeting-av-filters/csharp/MeetingAVFilter/ClientApp/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Sample app demonstrating meeting AV filters</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <!--
+      This is an sample app which demonstrates use of av filters in teams meeting.
```

Comment:
This is an sample app which demonstrates the use of AV filters in Teams meeting.